new_client/src/App.js

```diff
@@ -70,6 +71,7 @@ function App() {
               <ProtectedRoute path="/addevent" component={AddEvent} />
               <ProtectedRoute path="/event/edit/:id" component={AddEvent} />
               <Route path="/resources" component={ResourcePage} />
+              <Route path="/alumni" component={AlumniPage} />
```

Comment:
I know this isn't complete, but we use `lazy` for code splitting now, so maybe in a later commit, we can update this as well.